symfony/framework-bundle/7.2/public/index.php

```diff
@@ -2,6 +2,14 @@
 
 use App\Kernel;
 
+if (!is_dir(dirname(__DIR__).'/vendor')) {
+    throw new LogicException('Dependencies are missing. Try running "composer install".');
+}
+
+if (!is_file(dirname(__DIR__).'/vendor/autoload_runtime.php')) {
+    throw new LogicException('Symfony Runtime is missing. Try running "composer require symfony/runtime".');
+}
+
 require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
```

Comment:
I'd rather go with less calls on the hotpath, maybe something to sync with the console script?
```suggestion
if (false === include dirname(__DIR__).'/vendor/autoload_runtime.php') {
    throw new LogicException('Symfony Runtime is missing. Try running "composer require symfony/runtime".');
}
```

Reply:
I applied your suggestion here

but I am unsure to understand your second part of the comment : `maybe something to sync with the console script?`
may I ask you to explicit a bit more please ?